eng/performance/maui_scenarios_ios.proj

```diff
@@ -26,19 +27,19 @@
   </ItemDefinitionGroup>
 
   <ItemGroup>
-    <MAUIiOSScenario Include="Xamarin iOS Default $(RuntimeFlavor)">
+    <MAUIiOSScenario Include="Xamarin iOS Default $(RuntimeFlavor) HybridGlobalization $(HybridGlobalization)">
```

Comment:
We should think about the best way to add these properties going forward to ensure the best compatibility with using ADX and PowerBI for the scenario names. Specifically, I think if we could have the properties be additive to the name rather than added to every name, it would keep any PowerBI Maintenance and ADX queries that use test name from breaking each time we add another true or false property.

Reply:
That is right. 

> I think if we could have the properties be additive to the name rather than added to every name

Could you provide an example?

Alternative approach would be matching them by a unique value, such as primary key or something similar which is unique to scenarios. When a new scenario is introduced, a key is added to the database, and a config is updated on the CI.

Reply:
A good example of this making the naming additive is the name setup found here: https://github.com/dotnet/runtime/pull/89825/files#diff-cda44392aa226e93aee0bcef10f241925a7cb882ca107cef95a333e3ee1da8a6R35.